subsys/net/lib/sockets/sockets.c

```diff
@@ -331,6 +331,15 @@ static void zsock_accepted_cb(struct net_context *new_ctx,
 		k_condvar_init(&new_ctx->cond.recv);
```

Comment:
Please change the commit subject to `net: sockets: Fixes net_pkt leak in accept` and squash the commits to one. Commit msg should also have a body text (description why the patch exists) and signed-off-by line.

Reply:
The most recent PR should resolve all of the mentioned changes.